libs/clj-icat-direct/src/clj_icat_direct/queries.clj

```diff
@@ -517,21 +517,19 @@
 
 (def queries
   {:count-all-items-under-folder
-   "WITH user_groups AS ( SELECT g.*
+   "WITH user_groups AS ( SELECT g.group_user_id
                             FROM r_user_main u
                             JOIN r_user_group g ON g.user_id = u.user_id
                            WHERE u.user_name = ?
                              AND u.zone_name = ? ),
 
-         parent      AS ( SELECT * from r_coll_main
+         parent      AS ( SELECT DISTINCT coll_id, coll_name from r_coll_main
```

Comment:
@psarando it looks like in our actual databases, they are in fact unique. I think the query below needed it because of the join on `r_objt_access` which could result in multiple rows in the resultset.

It's still not enforced at a DB level, just higher up (presumably) in iRODS.